### PR TITLE
Release/v5.5.0: Changelog

### PR DESCRIPTION
**Pull request by ashnwade**
This PR addresses no issue.

### CODE REVIEW

File: changelog/5.5.0.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+## Released 13 August 2024
+
+## Gravwell
+
+### Additions
+
+* Added [multi-factor authentication (MFA)](/configuration/mfa/mfa) using TOTP and recovery codes.
+* Added the ability to specify <a href="/search/search.html#specifying-the-search-timeframe-in-query">time constraints</a> in the query itself.
+* Added a new [top/bottom](topbottom/topbottom) search module.
+* Added hinting for eval `in()` function.
```
Comment (david-fritz-gravwell) [inline changelog/5.5.0.md:12]:
```suggestion
* Added acceleration hinting for eval `in()` function.
```

File: changelog/5.5.0.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* Fixed an issue where a Dashboard tile could cover the settings button if live update was enabled.
+* Fixed an issue where the save button in Dashboard settings may cover other text.
+* Fixed an issue where SSO login may fail to redirect to home page for user that previously logged in and logged out. 
+* Fixed an issue with webserver startup when an indexer is down.
+* Fixed an issue with rapid unplanned ageout.
+* Fixed an issue with chart rendering in Systems & Health when browser is zoomed.
+* Improved performance for getting Systems & Health stats.
+* Improved display of Systems & Health calendar to better understand ingestion of historical data.
+* Improved validation for live update interval in Dashboards.
+* Prevented the current shard from aging out.
```
Comment (david-fritz-gravwell) [inline changelog/5.5.0.md:56]:
```suggestion
```

Comment (david-fritz-gravwell) [reply changelog/5.5.0.md:56]: This was always the case, but we broke it during 5.5.0 development.

**Review by david-fritz-gravwell — CHANGES_REQUESTED** [review]

**Review by john-floren-gravwell — DISMISSED** [review]

**Review by david-fritz-gravwell — DISMISSED** [review]
LGTM

**Review by evan-gravwell — DISMISSED** [review]

**Review by michael-wisely-gravwell — APPROVED** [review]
LGTM!

**Pull request merged**